src/main/java/com/parsing/mockdata/rozetka_parsing_item/LaptopItemMockDataGenerator.java

```diff
@@ -28,6 +32,17 @@ public class LaptopItemMockDataGenerator {
             "Huawei MateBook 14S 14.2\"", "Samsung Galaxy Book 2 Pro", "Huawei MateBook D 16"
     );
 
+    @PostConstruct
```

Comment:
Consider avoiding @PostConstruct, maybe static, or explain why @PostConstruct is necessary.